configure.ac

```diff
@@ -168,6 +168,18 @@ AS_IF([test "x$enable_abootsettings" != xno],
 AM_CONDITIONAL([WANT_ABOOTSETTINGS], [test x$enable_abootsettings != xno])
 
 
+#
+# Wheel is privileged
+#
+AC_ARG_ENABLE([usewheel],
+  [AS_HELP_STRING([--enable-usewheel],
+    [Users in wheel group are considered privileged])])
+
+AS_IF([test "x$enable-usewheel" != xno],
+  [AC_DEFINE([DSME_USEWHEEL], [1], [Users in wheel group are considered privileged])])
+
+AM_CONDITIONAL([WANT_USEWHEEL], [test x$enable-usewheel != xno])
```

Comment:
Looks like DSME_USEWHEEL ends up always defined regardless of configure options - which is not ok.

After taking a brief AC_ARG_ENABLE refresher... AFAICT enabling "disabled-by-default" config item (like this usewheel here) should use "x$topic" == xyes" type tests.

(On a quick glance a lot of the other already existing config options have similar issues, but fixing those falls outside the context of this pull req)